tokens/connect/ledger/moonbeam.md

```diff
@@ -92,4 +98,6 @@ And that is it! You've signed a transaction and sent some GLMR tokens using your
 
 ![MetaMask Ledger Allow Contracts Tx](/images/tokens/connect/ledger/moonbeam/ledger-11.png)
 
---8<-- 'text/disclaimers/third-party-content.md'
\ No newline at end of file
+--8<-- 'text/ledger/ledger-live.md'
```

Comment:
I think this line is also missing in corresponding file in PureStake/moonbeam-docs-cn repository PR#202.